### PR TITLE
[ERP-2002] Added basic operational status component and story.

### PR DESCRIPTION
**Pull request by NikuPAN**

### CODE REVIEW

File: src/components/OperationalStatus/OperationalStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  detected?: string;
+  error?: string;
+}
+
+interface Data extends Service {
+  last_updated?: string;
+  services: Service[];
+}
+
+export interface OperationalStatusData extends Data {
```
Comment (ryan-bennett) [inline src/components/OperationalStatus/OperationalStatus.tsx]: The convention we use for other components is to call this OperationalStatusProps, that way we know its the props for the OperationalStatus component

File: src/components/OperationalStatus/OperationalStatus.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { Meta, StoryFn } from "@storybook/react";
+import { OperationalStatus, OperationalStatusData } from "./OperationalStatus";
+import jsonData from "../../../public/status.json";
```
Comment (ryan-bennett) [inline src/components/OperationalStatus/OperationalStatus.stories.tsx]: To keep the story static, omit this line and embed the data with the story args, you can set up a Primary story and the the other story can deconstruct it's arg into theirs. Check Page.stories.tsx for an example

**Review by ryan-bennett — CHANGES_REQUESTED** [review]

File: src/components/OperationalStatus/OperationalStatus.tsx (hunk 1 above)
Comment (NikuPAN) [reply src/components/OperationalStatus/OperationalStatus.tsx]: Updated

File: src/components/OperationalStatus/OperationalStatus.stories.tsx (hunk 2 above)
Comment (NikuPAN) [reply src/components/OperationalStatus/OperationalStatus.stories.tsx]: Updated

**Review by ryan-bennett — APPROVED** [review]

**Pull request merged**